source/Calamari.Common/Plumbing/Deployment/PackageRetention/IJournal.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using Calamari.Common.Plumbing.Variables;
+using Octopus.Versioning;
+
+namespace Calamari.Common.Plumbing.Deployment.PackageRetention
+{
+    public interface IJournal
```

Comment:
I don't this broad interface name matches the quite specific contract it enforces.